ElasticFrameProtocol.cpp

```diff
@@ -167,6 +167,15 @@ ElasticFrameProtocolReceiver::unpackType1(const uint8_t *pSubPacket, size_t lPac
             pThisBucket->mActive = false;
             return ElasticFrameMessages::memoryAllocationError;
         }
+
+        if (pThisBucket->mOfFragmentNo < lType1Frame->hFragmentNo ||
```

Comment:
Is this check relevant for the other frame types?

Reply:
I have checked all std::copy_n and I think it looks OK for all package types.